src/art.rs

```diff
@@ -86,7 +86,7 @@ pub enum QueryType {
 /// - `Node48(Node48<P, Node<P, V>>)`: Represents an inner node with 256 keys and 48 children.
 /// - `Node256(Node256<P, Node<P, V>>)`: Represents an inner node with 256 keys and 256 children.
 ///
-pub(crate) enum NodeType<P: KeyTrait, V: Clone> {
+pub(crate) enum NodeType<P: KeyTrait, V: Clone + Ord> {
```

Comment:
Seems like adding the `Ord` trait bound for value types is a bit restrictive.

Reply:
Thank you for highlighting it. Could you let me know your thoughts on if the latest approach works?